target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz7490.dts

```diff
@@ -0,0 +1,282 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7490";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_info_red;
+		led-running = &led_power;
+		led-upgrade = &led_info_red;
+
+		led-dsl = &led_internet;
+		led-internet = &led_internet;
+		led-wifi = &led_wlan;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		// key "DECT"
+		key_dect {
+			label = "dect";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_PHONE>;
+		};
+		// key "WLAN/WPS"
+		key_wlan {
+			label = "wlan";
+			gpios = <&gpio 29 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		// led 5 "Power/DSL"
+		led_power: power {
+			label = "fritz7490:green:power";
+			gpios = <&gpio 45 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+		// led 4 "Internet"
+		led_internet: internet {
+			label = "fritz7490:green:internet";
+			gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
+		};
+		// led 3 "FixedLine"
+		led_fixedline: fixedline {
```

Comment:
The node label (led_fixedline) is only needed when you reference it. Please remove labels where not necessary.

Reply:
removed the labels